<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paradialog_littlequickwarp</class>
 <widget class="QDialog" name="Paradialog_littlequickwarp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_fileio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>451</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>File IO: </string>
   </property>
   <widget class="QLabel" name="label_img_sub_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input subject image: (to be warpped)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_sub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input subject marker file:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_img_sub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_img_sub">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>46</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_marker_sub">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>97</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_marker_sub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_marker_tar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_sub2tar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input target marker file:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_marker_tar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>146</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_img_warp">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>197</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_swc_grid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output warped image file:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_img_warp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_paras">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>451</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Paras:</string>
   </property>
   <widget class="QCheckBox" name="checkBox_padding">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>480</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Padding image before warping (only for comparing with JBA)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Image value interpolation method:</string>
    </property>
    <widget class="QRadioButton" name="radioButton_img_interp_nn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nearest Neighbor</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_img_interp_linear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Trilinear</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Displace field interpolation method:</string>
    </property>
    <widget class="QRadioButton" name="radioButton_df_interp_linear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Trilinear</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_df_interp_bspline">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>B-spline</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_resize">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Set warp image size different to subject image</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_Xdim">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Xdim">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Ydim">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Ydim">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Zdim">
    <property name="geometry">
     <rect>
      <x>273</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Zdim">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_paras</zorder>
  <zorder>buttonBox</zorder>
  <zorder>groupBox_fileio</zorder>
  <zorder>groupBox_resize</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Paradialog_littlequickwarp</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Paradialog_littlequickwarp</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
